workflow/tests/ACCA_Examples/Bob_Ross_Residence.xml

```diff
@@ -0,0 +1,1483 @@
+            </ExteriorShading>
+            <InteriorShading>
+              <SystemIdentifier id='WindowBedroom3WestInteriorShading'/>
+              <SummerShadingCoefficient>0.795</SummerShadingCoefficient>
+            </InteriorShading>
+            <Overhangs>
+              <Depth>1.5</Depth>
+              <DistanceToTopOfWindow>10.0</DistanceToTopOfWindow>
+              <DistanceToBottomOfWindow>12.0</DistanceToBottomOfWindow>
+            </Overhangs>
+            <AttachedToWall idref='WallBedroom3West'/>
+          </Window>
+          <Window>
+            <SystemIdentifier id='WindowBathroom2West'/>
+            <Area>5.0</Area>
+            <Orientation>west</Orientation>
+            <UFactor>0.57</UFactor>
+            <SHGC>0.56</SHGC>
+            <ExteriorShading>
+              <SystemIdentifier id='WindowBathroom2WestExteriorShading'/>
+              <SummerShadingCoefficient>0.8</SummerShadingCoefficient>
+            </ExteriorShading>
+            <InteriorShading>
+              <SystemIdentifier id='WindowBathroom2WestInteriorShading'/>
+              <SummerShadingCoefficient>0.795</SummerShadingCoefficient>
+            </InteriorShading>
+            <Overhangs>
+              <Depth>1.5</Depth>
+              <DistanceToTopOfWindow>10.0</DistanceToTopOfWindow>
+              <DistanceToBottomOfWindow>12.0</DistanceToBottomOfWindow>
+            </Overhangs>
+            <AttachedToWall idref='WallBathroom2West'/>
+          </Window>
+          <Window>
+            <SystemIdentifier id='WindowBedroom2West'/>
+            <Area>10.0</Area>
+            <Orientation>west</Orientation>
+            <UFactor>0.57</UFactor>
+            <SHGC>0.56</SHGC>
+            <ExteriorShading>
+              <SystemIdentifier id='WindowBedroom2WestExteriorShading'/>
+              <SummerShadingCoefficient>0.8</SummerShadingCoefficient>
+            </ExteriorShading>
+            <InteriorShading>
+              <SystemIdentifier id='WindowBedroom2WestInteriorShading'/>
+              <SummerShadingCoefficient>0.795</SummerShadingCoefficient>
+            </InteriorShading>
+            <Overhangs>
+              <Depth>1.5</Depth>
+              <DistanceToTopOfWindow>10.0</DistanceToTopOfWindow>
+              <DistanceToBottomOfWindow>12.0</DistanceToBottomOfWindow>
+            </Overhangs>
+            <AttachedToWall idref='WallBedroom2West'/>
+          </Window>
+        </Windows>
+        <Skylights>
+          <Skylight>
+            <SystemIdentifier id='SkylightFamilyRoom'/>
+            <Area>8.0</Area>
+            <Azimuth>0</Azimuth>
+            <UFactor>1.13</UFactor>
+            <SHGC>0.66</SHGC>
+            <AttachedToRoof idref='Roof1'/>
+            <AttachedToFloor idref='CeilingFamilyRoom'/>
+            <extension>
+              <Curb>
+                <Area>2.8</Area>
+                <AssemblyEffectiveRValue>1.96</AssemblyEffectiveRValue>
```

Comment:
It's very confusing that there are conflicting numbers everywhere, but the Word doc has clarifying language that the curb should be treated as insulated, not uninsulated as in the original spec. So I assume this should be 6.25.

Reply:
I know, but I think it clarified because there's no U value and SHGC lookup for uninsulated curb + insulated shaft skylight, so I'm using the values it gives (insulated curb + insulated shaft), but I'm not sure if we should specify the curb as insulated here.

Reply:
From the table 2B-3, if insulated, it's 1/0.15 = 6.67. I'll proceed with insulated for now.

Reply:
Ah right, there are slightly different U-factors for R-6 curb and R-6 shaft. 👍